### PR TITLE
post model implemented and tested

### PR DESCRIPTION
**Pull request by farishkash**

### CODE REVIEW

File: ruby-monday-blog/spec/models/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```
Comment (sarony) [inline ruby-monday-blog/spec/models/post_spec.rb]: You can actually shorten this to just `describe Post do` which is cleaner and easier to read.

File: ruby-monday-blog/spec/models/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  it "makes a new post with title" do
```
Comment (sarony) [inline ruby-monday-blog/spec/models/post_spec.rb]: I think the description should read "makes a new post with title and body" since it's testing for both. Also, I'm wondering if we should do a different test for the body. What do you think?

Comment (sarony) [inline ruby-monday-blog/spec/models/post_spec.rb]: A tool you might also want to check out is [Shoulda Matchers ](https://github.com/thoughtbot/shoulda-matchers). Instead of doing a full multi-line spec like this, it allows you to do one liners like `it { should have_attribute(:title) }` which is cleaner and easier to read for things like attributes. That way, you can focus the pages on most important specs like methods you wrote yourself, as opposed to testing methods given to you by Active Record.

**Comment by sarony** [comment]
@farishkash gave you some feedback on ways to make your spec a bit tighter. Take a look and let me know what you think :)

**Comment by farishkash** [comment]
@sarony I was thinking the same thing about making a separate test for body but I didn't know what is the standard procedure in that situation.  I will look into Shoulda Matchers, sounds like a good tool.

**Comment by farishkash** [comment]
Ok so I made changes to the test but I did not use Shoulda Matchers for this/

File: ruby-monday-blog/spec/models/post_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Post do
```
Comment (sarony) [inline ruby-monday-blog/spec/models/post_spec.rb]: I noticed you kept the `Rspec`. On purpose or missed it?

Comment (farishkash) [inline ruby-monday-blog/spec/models/post_spec.rb]: I thought you meant after the  describe not before.  Didn't realize you can leave that out.  I removed that now.

**Comment by sarony** [comment]
@farishkash looks good to me! Merge it in :)
